examples/client/client.c

```diff
@@ -1440,6 +1448,49 @@ static void Usage(void)
 #endif
 }
 
+static int SimulateWantWriteIOSendCb(WOLFSSL *ssl, char *buf, int sz, void *ctx)
+{
+    static int wantWriteFlag = 1;
```

Comment:
This has to be initialized to 1. If 0 instead, it won't exercise WANT_WRITE resilience in the handshake.